### PR TITLE
Add Tools panel to Settings dialog

### PR DESCRIPTION
**Pull request by mirka**
Based on #1109 

This makes the Import/Export tools reachable from a non-Electron environment.

screen shot 2018-12-21 at 1 19 08

### CODE REVIEW

File: lib/dialogs/settings/index.jsx [hunk 1]
```diff
@@ -141,6 +142,9 @@ export class SettingsDialog extends Component {
         <div className="dialog-column">
           <DisplayPanel buckets={buckets} />
         </div>
+        <div className="dialog-column">
```
Comment (roundhill) [inline lib/dialogs/settings/index.jsx]: Should we move these `dialog-column` divs to within the panel components? 🤔

File: lib/utils/export/index.js [hunk 2]
```diff
@@ -4,7 +4,9 @@ import { get } from 'lodash';
 import exportNotes from './export-notes';
 import exportToZip from './to-zip';
 
-const exportZipArchive = filename => {
+const filename = 'notes.zip';
```
Comment (roundhill) [inline lib/utils/export/index.js]: Just for fun, what if we called this `Simplenotes.zip`?

File: lib/dialogs/settings/index.jsx (hunk 1 above)
Comment (mirka) [reply lib/dialogs/settings/index.jsx]: True, the CSS is a bit messy! I refactored the TabPanels component so the CSS isn't all over the place.

File: lib/utils/export/index.js (hunk 2 above)
Comment (mirka) [reply lib/utils/export/index.js]: Maybe I'm stuffy and overprotective of the Simplenote® brand, but:

> NEVER modify a trademark by changing it to the plural form.

http://www.inta.org/trademarkbasics/factsheets/pages/trademarkusefactsheet.aspx

😂😂😂

**Pull request merged**